corehq/apps/es/cases.py

```diff
@@ -21,6 +21,9 @@
 
 class CaseES(HQESQuery):
     index = 'cases'
+    default_filters = {
+        'is_commcare_case': filters.doc_type("CommCareCase"),
```

Comment:
Hmm. It does appear like that's the case, but I ran into a couple of deleted cases when I was running this command. I don't have a great idea on why that is though. Possibly another bug or inconsistency in pillow top?